ui/urls.py

```diff
@@ -43,5 +43,6 @@
         r'vocabularies/(?P<vocab_slug>[-\w]+)/$',
         edit_vocabulary,
         name="edit_vocabulary"),
+    url(r'^search/', include('haystack.urls')),
```

Comment:
The URL is now just the "repositories/slug" URL. We're no longer importing any URLs from Haystack.